data/os/RedHat/8.yaml

```diff
@@ -0,0 +1,3 @@
+---
+profile_java::pkg_list:
+  "java-1.8.0-openjdk-headless":
```

Comment:
I question if the default version for RHEL8 should instead be either `java-11-openjdk-headless` or `java-17-openjdk-headless`. See https://access.redhat.com/articles/1299013#OpenJDK_Lifecycle_Dates_and_RHEL_versions

Reply:
...and https://openjdk.org/projects/jdk/

Reply:
We talked off line and are not going to address newer JDK yet.

Reply:
Note that I did also remove Hiera data (well, set to []) for RedHat/8.yaml. Java 1.8.0 is already getting installed, specifically on Ngale, and I don't know why. But I'd rather avoid adding it anywhere else that it's not present, if we start using this profile elsewhere soon, before we spend some more time hashing out what default makes sense for RHEL 8.